hadoop-ozone/dist/src/main/smoketest/s3/objectputget.robot

```diff
@@ -241,3 +260,98 @@ Create key twice with different content and expect different ETags
                                 Execute AWSS3Cli           rm s3://${BUCKET}/test_key_to_check_etag_differences
                                 Execute                    rm -rf /tmp/file1
                                 Execute                    rm -rf /tmp/file2
+
+*** Test Cases ***
```

Comment:
I think we can loop (`FOR` - `END`) over the buckets `${BUCKET}`, `${BUCKET1}` and type: OBS/FSO for reducing the redundancy under the individual test cases.